centers/models/region.py

```diff
@@ -7,6 +10,20 @@ class RegionFirstLayer(models.Model):
     def __unicode__(self):
         return unicode(self.name)
 
+    def centers(self):
```

Comment:
Intention is invisible.
I think centers it not a explicit name.
it must be meaningful